### PR TITLE
Added French translations in `i18n.js`.

### PR DESCRIPTION
**Pull request by alexsam29**
This PR is related to #33. I added French translations to `i18n.js`. It may not be perfect, but should do for now. Reviews by native French speakers are welcome.

### CODE REVIEW

**Review by AndrGab — APPROVED** [review]
Nice, Thank you very much

File: src/i18n.js [hunk 1]
```diff
@@ -221,6 +221,115 @@ const resources = {
...
+      patientName: "Nom du patient",
+      generalCondition: "Conditions générales",
+      bloodPressure: "Pression artérielle",
+      consciousnessLevel: "Niveau de conscience",
+      fever: "Fièvre",
+      respiration: "Respiration",
+      diuresis: "Diurèse",
+      observations: "Observations",
+      doctorName: "Nom du médecin",
+      signedAt: "Signé à",
```
Comment (devnullone) [inline src/i18n.js:254]: "SignedAt" in french give "Signé le" not "Signé à"

File: src/i18n.js [hunk 2]
```diff
@@ -221,6 +221,115 @@ const resources = {
...
+      add: "Ajouter",
+      normal: "Normal",
+      serious: "Sérieux",
+      stable: "Écurie",
```
Comment (devnullone) [inline src/i18n.js:258]: "stable" here I guess means constant so the right traduction in french is "stable". An "écurie" is where horses are living.

**Review by devnullone — COMMENTED** [review]
Hi guys, I put some reviews, hope that help :)

**Comment by AndrGab** [comment]
@devnullone Open a PR with that please 😃

**Pull request merged**